gc/base/ParallelHeapWalker.cpp

```diff
@@ -0,0 +1,162 @@
+/*******************************************************************************
```

Comment:
GlobalCollectorDelegate.*pp are new files that contain previously written (refactored) code, much of which was originally contributed from J9. The following files were migrated from (contributed by) J9:

MarkMapSegmentChunkIterator.*pp
ParallelHeapWalker.*pp
ParallelObjectheapIterator.*pp
ParallelGlobalGCTask.*pp